.github/actions/start-aws-runner/action.yml

```diff
@@ -41,14 +41,17 @@ runs:
         aws-region: us-east-2
     - name: Start EC2 runner
       id: start-ec2-runner
-      uses: supertopher/ec2-github-runner@base64v1.0.10
+      uses: airbytehq/ec2-github-runner@base64v1.1.0
       with:
         mode: start
         github-token: ${{ inputs.github-token }}
         ec2-image-id: ${{ inputs.ec2-image-id }}
         ec2-instance-type: ${{ inputs.ec2-instance-type }}
         subnet-id: ${{ inputs.subnet-id }}
         security-group-id: ${{ inputs.security-group-id }}
+        # this adds a label to group any EC2 runners spun up within the same action run
+        # this enables creating a pool of runners to run multiple/matrix jobs on in parallel
+        label: runner-pool-${{ github.run_id }}
```

Comment:
is this used within EC2 somewhere? what is the functional significance for adding this `label` property?

Reply:
It adds the label to the runner as seen by github ([UI here](https://github.com/airbytehq/airbyte/settings/actions/runners)). 

Labels we had before:
- [self-hosted, Linux, X64] on every runner
- a random unique label to _id_ a runner

This adds a new label that is unique to the github action run, meaning if we spin up 5 runners in the same action run, we now have a label that is shared across them all, giving us a pool of runners that we can use in the `runs-on: ` section of a job.